rocAL/rocAL/include/loaders/image/cifar10_data_loader.h

```diff
@@ -55,8 +55,8 @@ class CIFAR10DataLoader : public LoaderModule
     LoaderModuleStatus load_routine();
     std::shared_ptr<Reader> _reader;
     void *_dev_resources;
-    decoded_image_info _raw_img_info;       // image info to store the names. In this case the ID of image is stored in _roi_width field
-    decoded_image_info _output_decoded_img_info;
+    decoded_sample_info _raw_img_info;       // image info to store the names. In this case the ID of image is stored in _roi_width field
+    decoded_sample_info _output_decoded_img_info;
```

Comment:
Should the variables also needs to be changed as _raw_sample_info, _output_decoded_sample_info?

Reply:
cifar10 dataset only deals with images, so i think this name makes sense